### PR TITLE
Allow empty keybind

### PR DESCRIPTION
**Pull request by zethnest**
You can clear a Keybind with this PR.

### CODE REVIEW

File: arc-core/src/arc/input/KeyCode.java [hunk 1]
```diff
@@ -37,6 +37,7 @@
     //scroll
     scroll(KeyType.scroll, "Scrollwheel", true),
     //keyboard
+    nullKey(KeyType.key, "Empty"),
```
Comment (Anuken) [inline arc-core/src/arc/input/KeyCode.java:40]: There already is an 'unset' key.

**Comment by Anuken** [comment]
Why? The keybind dialog UI is already quite bloated.

File: arc-core/src/arc/input/KeyCode.java (hunk 1 above)
Comment (zethnest) [reply arc-core/src/arc/input/KeyCode.java:40]: But the unset key reverts it back to the default it seems.

**Comment by zethnest** [comment]
The problem I am having currently is, there are some controls that I'm sure I won't ever use.

Do you have other ideas in mind with clearing keys?

**Comment by Anuken** [comment]
That's a very strange problem. Why can't you just not use the keys? How are you accidentally hitting the backslash key?

**Comment by zethnest** [comment]
The backslash is positioned above the enter key. When I finish typing in chat and press enter, the fat finger sometimes hit it by accident after hitting enter. And then my UI is locked to block selection in Mindustry.

**Comment by Anuken** [comment]
I don't think this is necessary.

**Pull request closed without merging**